lib/rubocop/cop/mixin/safe_mode.rb

```diff
@@ -1,16 +1,25 @@
 # frozen_string_literal: true
 
+# rubocop:disable Style/ClassVars
+
 module RuboCop
   module Cop
     # Common functionality for Rails safe mode.
     module SafeMode
+      @@warned = false
+
       private
 
       def rails_safe_mode?
         safe_mode? || rails?
       end
 
       def safe_mode?
+        unless @@warned
+          @@warned = true
+          warn 'The `SafeMode` option will be removed in `RuboCop` 0.76. ' \
```

Comment:
I guess you can just move the warning to the body of the module, as I think everything in it is scheduled for removal.

Reply:
Good idea! Updated.